Src/Sarif.PatternMatcher/AnalyzeOptions.cs

```diff
@@ -41,5 +41,9 @@ public class AnalyzeOptions : AnalyzeOptionsBase
             HelpText = "The maximum file size (in kilobytes) that will be analyzed.",
             Default = 1024)]
         public int FileSizeInKilobytes { get; internal set; }
+
+        [Option(
+            "disable-validation-caching")]
```

Comment:
>disable-validation-caching [](start = 13, length = 26)

disable-dynamic-validation-caching

DisableDynamicValidationCaching

Reply:
renamed

---
In reply to: [592834129](https://github.com/microsoft/sarif-pattern-matcher/pull/288#discussion_r592834129) [](ancestors = 592834129)